server/models.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const UserSchema = mongoose.Schema(
+    {
+        username: {
+            type: String,
+            required: true,
+        },
+        email: {
+            type: String,
+            required: true,  
+        },
+        password : {
+            type: String,
+            required: true,
+        }
+    }
+);
+
+const User = mongoose.model('User', UserSchema); 
+
+const EntrySchema = mongoose.Schema(
+    {
+        userId: {
+            type: mongoose.Schema.Types.ObjectId, 
+            required: true,
+        },
+        entryText: {
+            type: String,
+            required: true,
+        },
+    },
+    {
+        timestamps: true,
+    }
+);
+
+const Entry = mongoose.model('Entry', EntrySchema);
+
+module.exports = {Entry, User};
```

Comment:
It's good to be a bit more modular with the code here; let's split this into two files called `post.js` and `user.js` under a folder called `models`

Reply:
Also rename "entries" to "posts"